### PR TITLE
Remove mongo refs

### PR DESCRIPTION
**Pull request by jkerr321**
Fix for https://response.ftops.tech/incident/854/

mLab add on has been removed from Heroku. This app initialises with Mongo and therefore crashes immediately. This app is now (in theory) only used for article rendering, which doesn't require Mongo.

I've removed Mongo references in the app to the point that it runs locally and successfully renders an article

The first set of mongo-removing-changes were made in this PR - https://github.com/Financial-Times/alphaville-blogs/pull/87

### CODE REVIEW

File: bin/www [hunk 1]
```diff
@@ -48,6 +48,16 @@ function start() {
 
 	const server = http.createServer(app);
 
+	/**
+	 * Listen on provided port, on all network interfaces.
+	 */
+
+	server.listen(port, function () {
```
Comment (lucas42) [inline bin/www:55]: Ah yes - listening for incoming traffic is really useful!  😂

**Review by binaryberry — APPROVED** [review]
LGTM! Please do amend the readme later :-)

**Review by lucas42 — APPROVED** [review]
Well done on working out what was wrong!

The previous PR also removed the call to `sortArticles`. Does this need restoring too?  Or is it no longer needed?

**Comment by jkerr321** [comment]
no longer needed @lucas42

**Comment by lucas42** [comment]
Cool - looks like nothing is calling that function anymore.  Might be worth a tidy up at some point.  (If the plan is for this app to stick around)

**Comment by jkerr321** [comment]
Yep - this whole app will be cleaned up with the final part of the migration for Alphaville - almost all of it will become unnecessary  🙌

**Pull request merged**